### PR TITLE
Added a category for feature toggle libraries

### PR DESCRIPTION
**Pull request by tompave**
## Feature Flags and Toggles: new category

I've added a new "Feature Flags and Toggles" section to the list.

These two packages have been added:
* `flippant`
* `fun_with_flags` (disclaimer: I'm the author)

Also, the `molasses` library was already present in the "Miscellaneous" section and I've just moved it.

The packages are listed in alphabetic order.

I could find a couple of other libraries serving the same purpose in hex.pm, but I've only included the ones that have proper test coverage and that seem to be maintained.

This PR would solve issues https://github.com/h4cc/awesome-elixir/issues/3780 and https://github.com/h4cc/awesome-elixir/issues/2614.

### CODE REVIEW

**Comment by tompave** [comment]
Hi @h4cc, have you had a chance to review this?

File: README.md [hunk 1]
```diff
@@ -552,6 +553,14 @@ There are [other sites with curated lists of elixir packages](#other-awesome-lis
...
 * [weather](https://github.com/tacticiankerala/elixir-weather) - A command line weather app built using Elixir.
 
+## Feature Flags and Toggles
+*Libraries to manage feature toggles (AKA feature flags): ON/OFF values that can be toggled at runtime through some interface*
+
+* [flippant](https://github.com/sorentwo/flippant) - Feature flipping for the Elixir world.
+* [fun_with_flags](https://github.com/tompave/fun_with_flags) - A feature toggle library using Redis for persistance, ETS for local caching and Redis PubSub for inter-node cache busting. Optimized for speed and low Redis usage.
+* [molasses](https://github.com/securingsincity/molasses) - A feature toggle library using redis or SQL (using Ecto) as a backing service.
+
+
```
Comment (fladson) [inline README.md]: extra new line 😁

**Comment by tompave** [comment]
@fladson Thanks, fixed it!
So, what do we need to do to get feedback from @h4cc?

**Comment by fladson** [comment]
If he can't review it in a few days I will merge it. 😁

**Comment by tompave** [comment]
Thanks! 👍

**Pull request merged**